src/screens/cryptocurrency/CryptocurrencyInfo.tsx

```diff
@@ -0,0 +1,153 @@
+import CirclePlusSmallIcon from "@assets/icons/circle-plus-small.svg";
+import { ActivityIndicator, StyleSheet, View } from "react-native";
+import { Tabs } from "react-native-collapsible-tab-view";
+
+import Details from "./tabs/Details";
+import Informations from "./tabs/Informations";
+
+import Container from "@/components/Container";
+import CryptocurrencyHeader from "@/components/CryptocurrencyHeader";
+import Button from "@/components/buttons/Button";
+import CustomTabBar from "@/components/navigation/CustomTabBar";
+import Label from "@/components/typography/Label";
+import { accent, dark, spacing } from "@/constants/DesignTokens";
+import useGetTokenInfo from "@/hooks/useGetTokenInfo";
+
+type CryptocurrencyInfoProps = {
+  cryptocurrency: string;
```

Comment:
It should be capitalized

```suggestion
  cryptoCurrency: string;
```